pymagicc/__init__.py

```diff
@@ -1,6 +1,11 @@
 # Copyright (c) 2017 pymagicc authors:
-#   Robert Gieseke <robert.gieseke@pik-potsdam.de>
-# Free software under GNU Affero General Public License v3, see LICENSE
+#   - Robert Gieseke <robert.gieseke@pik-potsdam.de>
```

Comment:
Let's do it the Pandas way maybe?
```
Copyright (c) 2008-2011, AQR Capital Management, LLC, Lambda Foundry, Inc. and PyData Development Team
Copyright (c) 2011-2020, Open source contributors.
```
https://github.com/pandas-dev/pandas/blob/master/LICENSE

```
 Pymagicc developers and contributors
```

I think it's not even necessary to have this in the code file, Pandas hasn't either.

Reply:
Updated in c37c7b2